src/components/inputs/_input.scss

```diff
@@ -224,6 +224,8 @@
   margin-left: 5rem;
 }
 
-div[class*='-openmrs-esm-ohri-covid___input__formField___3aqHt'] {
-  margin-right: 20rem !important;
+.formField :global(.cds--form-item){
+  margin-right: 02rem !important;
+  flex: none !important;
+  box-sizing: content-box !important;
 }
```

Comment:
@ODORA0 I'm having a bad feeling about this change (The CSS selector to be more specific). You're using a global selector and it can potentially affect all form items (`.cds--form-item`). Can you push up the specificity to only target the date control?

Reply:
@samuelmale Mmh thinking about it, something like this should suffice then
`.formField > div {
`

Reply:
Jonathan, can you only target the date widget? You can use a different CSS class that only applies for the date control. That styling applies to all div of parents whose class is `formField`. That is problematic